src/modules/user-groups/services/user-group-population-query.service.spec.ts

```
@@ -1,9 +1,11 @@
+import { NotFoundException } from '@nestjs/common';
 import { TransactionService } from '@new-hros/libs-sql';
 
 import { UserGroupMatchingEngine } from './user-group-matching.engine';
 import { UserGroupPopulationQueryService } from './user-group-population-query.service';
 import { MatchingRule } from '../domain/value-objects/matching-rule.vo';
 import { UserGroup } from '../entities/user-group.entity';
+import { UserGroupMembership } from '../entities/user-group-membership.entity';
```

ESLint

[error] `../entities/user-group-membership.entity` import should occur before import of `../entities/user-group.entity`

(import/order)